### ISSUE TITLE
Black Screen

### ISSUE THREAD

**Issue by archon808** [federico-mammano/Oculus-SDK-0.4.4-beta-OpenGL-Demo#3]
I get a black screen when running the opengl tiny room project. I also tried commenting out the "OPENGL DIRECT MODE" but the screen is still black. I have a NVidia GTX645.

What should I try to fix it?

**Comment by archon808** [comment]
the first of many similar errors when run through gdebugger : 

Debug String: Detected error: The debugged process asked for an extension function pointer (wglChoosePixelFormatARB) from one render context, but called this function pointer in another render context (context #0)

**Comment by federico-mammano** [comment]
The function pointers are based on the current context. As a result, it is possible to get different function pointers for different OpenGL contexts. 
gDEBugger is just telling you that it has detected that you're using function pointers with a different context from the original used to acquire them. 
However, as I know, it should work in any case. It will not work if the contexts aren't from the same vendor or GPU or SLI/CrossFire GPU setup.

Are you able to see something on your HMD?

**Comment by archon808** [comment]
Thanks for the reply.
I was not able to get your project to work. I only see black screen.
Another person on your comment section also complain of a black screen.
I find it interesting that he has a NVidia 665 or something like that, and I have a 645. Very similar, so thats possibly a hint as to the issue. maybe driver, or maybe your style of code, or function choice is no longer supported. 

I don't think the problem is the driver, and I dont think it is with the dk2 library, because the Tuscany demo using opengl works perfect, so I know its possible. 

Anyways if you have the fix, I would love to see it.  Please.... !!!

Date: Sat, 21 Mar 2015 06:59:41 -0700

**Comment by elect86** [comment]
Hi,

I have also a black screen

this my console

http://pastebin.com/AYv4cMSw

**Comment by elect86** [comment]
Debug: [HSWDisplay] Display()
Debug: [HSWDisplay GL] DisplayInternal()
Error: [From Service] {ERR-009w} [HIDDevice] SetFeatureReport 17 failed
Error: [From Service] {ERR-010w} [HIDDevice] GetFeatureReport 2 failed
Error: [From Service] {ERR-027} [WatchDogObserver] Deadlock detected: LongPoll
[SensorStateReader] Prediction interval too high: 0.108387 s, clamping at 0.100000 s
Debug: [NetClient] Disconnected

**Comment by archon808** [comment]
Like I said, I gave up on this project (as it is both un-operational, and unsupported), and instead started with the Oculus Tuscany demo which has a robust OpenGL path that, depending on the computer, works with both direct to rift, and extended desktop modes.

**Comment by elect86** [comment]
yeah, no problem, anyway I am using it as a base to update my jogl roomTiny sample to 0.4.4, so the most important thing is that it was running, at least on your machine..

**Comment by elect86** [comment]
Ciao Federico,

I noticed a mistake, in the main loop when you call ClearAndSetRenderTarget(), the DX version pass a fbo image with both color and depth attachment..

You pass, instead, a fbo containing only a color attachment... you dont use the depth buffer at all..


elect
